app/controllers/notes/chapters/chapter_notes_controller.rb

```diff
@@ -10,7 +10,7 @@ def new
       def create
         @chapter_note = chapter.chapter_note.build(chapter_note_create_params.to_h)
 
-        if @chapter_note.save
+        if @chapter_note.valid? && @chapter_note.save
```

Comment:
it is not `@chapter_note.save` enough? 
I think it triggers the valid?

Reply:
https://github.com/remi/her#validations - docs say otherwise?